Description: add multiarch support
 Adds multiarch support.
Bug: https://bugs.launchpad.net/ubuntu/+source/sni-qt/+bug/838470
Author: Praveen Arimbrathodiyil <pravi.a@gmail.com>
Last-Update: 01-10-2011
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -56,7 +56,7 @@
 configure_file(dbusmenu-qt.pc.in ${CMAKE_BINARY_DIR}/dbusmenu-qt.pc @ONLY)
 
 install(FILES ${CMAKE_BINARY_DIR}/dbusmenu-qt.pc
-    DESTINATION lib${LIB_SUFFIX}/pkgconfig
+    DESTINATION lib/${CMAKE_LIBRARY_ARCHITECTURE}/pkgconfig
     )
 
 add_subdirectory(src)
--- a/dbusmenu-qt.pc.in
+++ b/dbusmenu-qt.pc.in
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=@CMAKE_INSTALL_PREFIX@
-libdir=@CMAKE_INSTALL_PREFIX@/lib
+libdir=@CMAKE_INSTALL_PREFIX@/lib/@CMAKE_LIBRARY_ARCHITECTURE@
 includedir=@CMAKE_INSTALL_PREFIX@/include/dbusmenu-qt
 
 Name: libdbusmenu-qt
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -73,7 +73,7 @@
     )
 
 install(TARGETS dbusmenu-qt
-    LIBRARY DESTINATION lib${LIB_SUFFIX})
+    LIBRARY DESTINATION lib/${CMAKE_LIBRARY_ARCHITECTURE})
 
 install(DIRECTORY .
     DESTINATION include/dbusmenu-qt
